src/main/java/io/confluent/connect/jdbc/JdbcUtils.java

```diff
@@ -131,7 +131,8 @@ public static String getAutoincrementColumn(Connection conn, String table) throw
     log.trace("Falling back to SELECT detection of auto-increment column for {}:{}", conn, table);
     Statement stmt = conn.createStatement();
     try {
-      rs = stmt.executeQuery("SELECT * FROM \"" + table + "\" LIMIT 1");
+      String quoteString = getIdentifierQuoteString(conn);
+      rs = stmt.executeQuery("SELECT * FROM " + quoteString + table + quoteString + " LIMIT 1");
```

Comment:
At the moment, these are all coming from a query against the database so there shouldn't be an issue. When I add table whitelist/blacklists, there will be user input, but that's a trusted user (you're also presumably giving the connection info, including password, via the same set of configs).

In general, while I think there are definitely security concerns in connectors, it's unlikely to be an issue in the first iteration of any connector since there are really only a small number of agents involved (Kafka, Kafka Connect, the other system, and the user submitting the connector).